### ISSUE TITLE
Unable to open ccsm

### ISSUE THREAD

**Issue by easdbe** [compiz-reloaded/ccsm#2]
Traceback (most recent call last):
  File "/usr/bin/ccsm", line 30, in <module>
    import gi
ImportError: No module named gi

**Comment by raveit65** [comment]
sorry, i can't help....my crystal ball i dead.
version?
distro?

**Comment by soreau** [comment]
I'm guessing you need to install python-gi or the relevant python gobject introspection package for your distro

**Comment by easdbe** [comment]
Debian testing/Python 2.7.11+

Installed python-gi and cssm now returns

```
Traceback (most recent call last):
  File "/usr/bin/ccsm", line 33, in <module>
    gi.require_version('Gtk', '3.0')
  File "/usr/lib/python2.7/dist-packages/gi/__init__.py", line 106, in require_version
    (namespace, version))
ValueError: Namespace Gtk not available for version 3.0
```

**Comment by XRevan86** [comment]
Not a bug. See description for the [0.8.12 release](https://github.com/compiz-reloaded/ccsm/releases/tag/v0.8.12).
